crm_lead_code/models/crm_lead.py

```diff
@@ -9,8 +9,8 @@
 class CrmLead(models.Model):
     _inherit = "crm.lead"
 
-    code = fields.Char(
-        string='Lead Number', required=True, default="/", readonly=True)
+    code = fields.Char(string='Lead Number', required=True, default="/",
+                       readonly=True)
```

Comment:
It' just my personal opinion, but I prefer the former formatting.
In general, either we like or dislike the formatting style, we should avoid editing lines just to change to another also valid formatting choice.